scripts/zones/Bostaunieux_Oubliette/npcs/Novalmauge.lua

```diff
@@ -109,5 +108,6 @@ function onEventFinish(player, csid, option, npc)
         tpz.wsquest.handleEventFinish(wsQuest, player, csid, option, ID.text.SPIRAL_HELL_LEARNED)
     end
 
-    npc:wait(0)
+    npc:speed(12)
```

Comment:
If the "new" initNpcAi (which has a commented TODO to change the name) ends up setting a local var to the NPC, then could add a `tpz.path.resume()` which sets the speed back and calls `npcBasicPath`

Reply:
sounds good to me, seems a better way of doing it.

Reply:
Might aswell hold this till that gets implimented if thats what you want to do, then i can change it, might have to give me a nudge to change it incase i forget lol.

Reply:
implimented pathStop() and pathResume() to move away from using wait()